eclipse-collections/src/main/java/org/eclipse/collections/impl/map/mutable/UnifiedMap.java

```diff
@@ -2145,6 +2149,26 @@ public boolean isEmpty()
             return UnifiedMap.this.isEmpty();
         }
 
+        @Override
+        public K getFirst()
+        {
+            if (UnifiedMap.this.size() > 0)
+            {
+                return (K) this.toArray()[0];
+            }
+            return null;
+        }
+
+        @Override
+        public K getLast()
+        {
+            if (UnifiedMap.this.size() > 0)
+            {
+                return (K) this.toArray()[UnifiedMap.this.size() - 1];
```

Comment:
@johnjstuart I don't think that convering to array just to get the last element is a good idea, it's too costly IMHO

Reply:
@johnjstuart use `IterableIterate.getLast(this)`?

Reply:
Done 👍